pilot/pkg/model/config.go

```diff
@@ -318,6 +324,22 @@ const (
 
 	// IstioMeshGateway is the built in gateway for all sidecars
 	IstioMeshGateway = "mesh"
+
+	// https://openid.net/specs/openid-connect-discovery-1_0.html
+	// OpenID Providers supporting Discovery MUST make a JSON document available at the path
+	// formed by concatenating the string /.well-known/openid-configuration to the Issuer.
+	openIDDiscoveryCfgURLSuffix = "/.well-known/openid-configuration"
```

Comment:
these should all to into the auth_cache.go

Reply:
done